pkg/frontend/clustermanager_get_test.go

```diff
@@ -0,0 +1,153 @@
+package frontend
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	v20220904 "github.com/Azure/ARO-RP/pkg/api/v20220904"
+	"github.com/Azure/ARO-RP/pkg/metrics/noop"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func TestGetClusterManagerConfiguration(t *testing.T) {
+	ctx := context.Background()
+
+	mockSubscriptionId := "00000000-0000-0000-0000-000000000000"
+	resourcePayload := "eyAKICAiYXBpVmVyc2lvbiI6ICJoaXZlLm9wZW5zaGlmdC5pby92MSIsCiAgImtpbmQiOiAiU3luY1NldCIsCiAgIm1ldGFkYXRhIjogewogICAgIm5hbWUiOiAic2FtcGxlIiwKICAgICJuYW1lc3BhY2UiOiAiYXJvLWY2MGFlOGEyLWJjYTEtNDk4Ny05MDU2LWYyZjZhMTgzN2NhYSIKICB9LAogICJzcGVjIjogewogICAgImNsdXN0ZXJEZXBsb3ltZW50UmVmcyI6IFtdLAogICAgInJlc291cmNlcyI6IFsKICAgICAgewogICAgICAgICJhcGlWZXJzaW9uIjogInYxIiwKICAgICAgICAia2luZCI6ICJDb25maWdNYXAiLAogICAgICAgICJtZXRhZGF0YSI6IHsKICAgICAgICAgICJuYW1lIjogIm15Y29uZmlnbWFwIgogICAgICAgIH0KICAgICAgfQogICAgXQogIH0KfQo="
+
+	type test struct {
+		name            string
+		ocmResourceType string
+		ocmResourceName string
+		clusterName     string
+		apiVersion      string
+		fixture         func(*testdatabase.Fixture, *test, string)
+		wantStatusCode  int
+		wantResponse    *v20220904.ClusterManagerConfiguration
+		wantError       string
+	}
+	createSingleDocument := func(f *testdatabase.Fixture, tt *test, resourceKey string) {
```

Comment:
Could we separate these `createSingleDocument` vars out from the individual test functions and re-use them?

Reply:
it seems like a DRY principal we could leverage here :+1: